main.py

```diff
@@ -1,5 +1,5 @@
 def writeMsg(a,b):
-    print(a+b)
+    print(a+b+c+d)
 
 # call the function
-writeMsg(1,2)
\ No newline at end of file
+writeMsg(1,2,3,4)
```

Comment:
The argument list to this function is getting quite long. It should instead take a dataset.